apis/submariner/v1alpha1/submariner_types.go

```diff
@@ -96,6 +97,13 @@ type DaemonSetStatus struct {
 	MismatchedContainerImages bool                     `json:"mismatchedContainerImages"`
 }
 
+type DeploymentInfo struct {
+	KubernetesType        KubernetesType `json:"kubernetesType,omitempty"`
+	KubernetesTypeVersion string         `json:"kubernetesTypeVersion,omitempty"`
+	KubernetesVersion     string         `json:"kubernetesVersion,omitempty"`
```

Comment:
What's the difference between those two?

Reply:
As explained in slack it's just for example the OCP version vs K8s version,although those should go hand in hand, and easy to co-relate, it could be useful to report them here in some cases.